src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
         if (logger.isTraceEnabled())
         {
             logger.trace(diagnosticContext
-                .makeTimeSeriesPoint("incoming_packet", nowMs)
+                .makeTimeSeriesPoint("in_pkt", nowMs)
```

Comment:
just wondering, what's the reason for the shorter name>

Reply:
The motivation here is to save some bytes.. These are the packet trace logs that need to be captured so I'm trying to save whatever I can in terms of size.

Reply:
ok, i think the way they are now is a bit ambiguous, in fact i'd missed here that this was referring to a timestamp.  i'd say keep them legible enough that there isn't ambiguity, or, go all the way and make them totally encoded (and short) and create a script or something that will be used to parse them?

Reply:
i could see this be `in_pkt_ts`

Reply:
I think you had it right, in_pkt is the name of the measurement. The second parameter sets the timestamp of the measurement.